lib/v2/datasource/remote/api/profile_repository.dart

```diff
@@ -5,16 +5,18 @@ import 'package:seeds/v2/datasource/remote/api/eos_repository.dart';
 import 'package:seeds/v2/datasource/remote/api/network_repository.dart';
 import 'package:seeds/v2/datasource/remote/model/profile_model.dart';
 import 'package:seeds/v2/datasource/remote/model/transaction_response.dart';
+import 'package:seeds/v2/datasource/settings_storage.dart';
 
 export 'package:async/src/result/result.dart';
 
 class ProfileRepository extends NetworkRepository with EosRepository {
-  Future<Result> getProfile(String userAccount) {
-    print('[http] get seeds getProfile $userAccount');
-
+  Future<Result> getProfile() {
+    final accountName = settingsStorage.accountName;
+    print('[http] get seeds getProfile $accountName');
+    // TODO(Raul): Remove this en-point with settingsStorage.nodeEndpoint idk why but when I do it throws an error.
```

Comment:
we cant forget about this. Please create a story to come back and take care of this and add the story number to the todo

Reply:
Okay